contracts/seed/SeedFactory.sol

```diff
@@ -72,17 +72,31 @@ contract SeedFactory is CloneFactory {
         avatar = _newAvatar;
     }
 
+    /**
```

Comment:
if this is just a dup of the docs in Seed, I think we should fix these issues there too.